.github/workflows/build.yml

```
@@ -115,6 +116,69 @@ jobs:
           leeway build dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}"
           echo "previewctl_hash=$(leeway describe dev/preview/previewctl:docker -Dversion="${{needs.configuration.outputs.version}}" -t '{{ .Metadata.Version }}')" >> $GITHUB_OUTPUT
 
+  update_pr_desc:
+    name: Update PR summary
+    needs: [ configuration ]
+    if: needs.configuration.outputs.pr_number != ''
```

Once we replace this the first time, we don't need to run it anymore. Expose an output `pr_body` in the same way as the number.

```suggestion
    if: needs.configuration.outputs.pr_number != '' && contains(needs.configuration.outputs.pr_body, 'gitpod:summary')
```